static/app/components/metrics/mriSelect/index.tsx

```diff
@@ -214,6 +216,19 @@ export const MRISelect = memo(function MRISelect({
     [onChange]
   );
 
+  const maxLength = useMemo(() => {
+    if (breakpoints.small) {
+      // at least small size screen, no problem with fitting 100 characters
+      return 100;
+    }
+    if (breakpoints.xsmall) {
+      return 35; // at least xsmall size screen, no problem with fitting 35 characters
+    }
+
+    // screen smaller than xsmall, 22 characters is optimal
+    return 22;
```

Comment:
please let's change it to '20' so the string fits for the iPhone see as well 

![image](https://github.com/user-attachments/assets/ef686929-9069-43d5-b311-d8fe501a7db0)

![image](https://github.com/user-attachments/assets/d4af21e3-0cb1-4770-a653-4322f080912f)